docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: '2'
```

Comment:
This needs to be `docker-compose.yml.dist`. The YML file needs to be gitignored. Please check the other components as an example.

Reply:
Then you need to sync in with @ademarco or @pfrenssen, apparently, you just have to create the file: docker-compose.override.yml to override the current setup.

Reply:
Indeed we should start moving towards the following: https://docs.docker.com/compose/reference/overview/#specifying-multiple-compose-files 

Not sure if the README.md of this module has been updated though.

Reply:
Let's create a follow up issue where we change this on all our components and update the README.md too. @upchuk @voidtek @upchuk could one of you do this please? I'm not at work today (as you can see :smile:) Thanks!

Reply:
Done.